app/Modules/ExpertPanel/Models/ExpertPanel.php

```diff
@@ -199,37 +196,6 @@ public function evidenceSummaries()
         return $this->hasMany(EvidenceSummary::class);
     }
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
```

Comment:
This does not appear to be used anywhere in the codebase, the model `LeadershipAttestation` does not exist, and the table in the dump you recently sent me is empty.